static_src/components/users.jsx

```diff
@@ -163,15 +164,13 @@ export default class Users extends React.Component {
 
   get userInvite() {
     if (!this.currentUserIsOrgManager) {
-      const cfAPILink = 'https://cloud.gov/docs/getting-started/setup/#set-up-the-command-line';
-
       return (
         <PanelDocumentation>
-          {`Currently, only an org manager can invite users to this ${this.entityType}
+          Currently, only an org manager can invite users to this { this.entityType }
           via the dashboard. If the user you want to add is already a cloud.gov member,
-          you can invite them using the <a href=${cfAPILink}>cloud foundry api. </a>
-          Speak to your org manager if you need to add a user to this ${this.entityType} who
-          is not a member of cloud.gov`}
+          you can invite them using the <a href={ cfAPILink }>cloud foundry api. </a>
+          Speak to your org manager if you need to add a user to this { this.entityType } who
+          is not a member of cloud.gov
```

Comment:
not necessary but it would be helpful if we could inject the actual name of the organization so it would be `is already a member of the "test" organization` (given the current org is `test`)